source/beamline/VESPERS/VESPERSBeamline.cpp

```diff
@@ -1243,7 +1249,7 @@ void VESPERSBeamline::openValve(int index)
 {
 	if (index >= 0 && index < valveSet_->count()){
```

Comment:
Again, this function will work for sure.
But the index should start from 1, since index 0 is `vvrFE1`,which is an instance of `AMReadOnlyPVwStatusControl`.